lib/lens.ex

```diff
@@ -17,12 +17,14 @@ defmodule Lens do
   end
 
   @doc ~S"""
-  Returns a lens that focuses on the whole data.
+  Returns a lens that focuses on all of the data in the given scope.
```

Comment:
But "all of the data" by itself is no less confusing. The problem which confused me was exactly that it wasn't _all of the data_ (which would have been the the `root` of the whole data structure), but rather the subtree that is currently focused on.

Do you have a suggestion for a better wording?